api/turing/validation/validator.go

```diff
@@ -129,6 +129,28 @@ func checkTrafficRuleName(sl validator.StructLevel, fieldName string, value stri
 	}
 }
 
+func checkDefaultTrafficRule(
+	sl validator.StructLevel,
+	fieldName string,
+	defaultTrafficRule *models.DefaultTrafficRule,
+) {
+	defaultTrafficRuleDescription := strings.Join([]string{
+		"Since 1 or more Custom Traffic rules have been specified,",
+		"a default Traffic rule is required.",
+	}, " ")
+	if defaultTrafficRule == nil {
+		sl.ReportError(defaultTrafficRule, fieldName, "defaultTrafficRule", defaultTrafficRuleDescription, "")
+	}
+}
+
+func checkTrafficRuleConditions(sl validator.StructLevel, fieldName string, rule *models.TrafficRule) {
```

Comment:
We don't need this anymore if putting back in `notBlank` in `api/turing/models/traffic_rules.go`, right?

Reply:
Same reason for why we cannot do so - https://github.com/caraml-dev/turing/pull/232#discussion_r964668455.

Reply:
This function has been removed.